google_analytics/preprocessing.py

```diff
@@ -161,6 +161,39 @@ def preprocess_and_save(data_dir, nrows_train=None, nrows_test=None, start_x_tra
     x_test.to_csv(os.path.join(data_dir, "preprocessed_x_test.csv"), index=False, encoding="utf-8")
 
 
+def add_isVisiting(x, y=None, y_visit_col_name=None, is_testing_set=True):
```

Comment:
I would give no default value to `is_testing_set` so that the user is always aware of what is happening because he/she has to set it explicitly. Now the user might call it without specifying the boolean and get unexpected results. But still that's a minor point.